superset-frontend/plugins/plugin-chart-table/src/TableChart.tsx

```diff
@@ -317,7 +317,7 @@ export default function TableChart<D extends DataRecord = DataRecord>(
   const getColumnConfigs = useCallback(
     (column: DataColumnMeta, i: number): ColumnWithLooseAccessor<D> => {
       const { key, label, isNumeric, dataType, isMetric, config = {} } = column;
-      const isFilter = !isNumeric && emitFilter;
+      const isFilter = emitFilter;
```

Comment:
is this variable needed now? (meaning, can we use `emitFilter` directly?)

Reply:
yes, sure.